src/ngamsCore/setup.py

```diff
@@ -30,7 +30,7 @@
             break
 
 # We definitely require this one
-install_requires = ['DBUtils', 'six>=1.10', 'requests']
+install_requires = ['DBUtils', 'six>=1.9', 'requests']
```

Comment:
I had 1.10 because there was a race condition in the mechanism used by `six.moves` that (I thought) had been fixed in that version, but it turns out the fix was put in 1.9: https://github.com/benjaminp/six/blob/master/CHANGES. So yes, let's go with 1.9.